### ISSUE TITLE
add image background

### ISSUE THREAD

**Issue by davestaab** [KnoxDevs/knoxdevs-website#2]

**Comment by Marviel** [comment]
@davestaab do you have any ideas for this, such as picture, etc? does this hurt the eyes currently?

**Comment by davestaab** [comment]
@Marviel hey, sorry for the slow response. I made this because there was some discussion in the slack channel about background images. Alex offered some pictures I believe. I just put this in so we'd remember to add it. 

I don't have any problems with the current design. I think it's coming along nicely. Keep up the good work! :+1:

**Comment by Marviel** [comment]
Yeah I guess we do need to look into it; thanks for pointing it out. :smile:
